Reach/Payment/Model/ResourceModel/Carrier/CsvHsCode/DataHashGenerator.php

```diff
@@ -17,6 +17,7 @@ public function getHash(array $data)
         $id = $data['id'];
         $sku = $data['sku'];
         $hsCode = $data['hs_code'];
-        return sprintf("%s-%d-%s-%F", $id, $sku, $hsCode);
+        $countryOfOrigin = $data['country_of_origin'];
+        return sprintf("%s-%d-%s-%F", $id, $sku, $hsCode, $countryOfOrigin);
```

Comment:
why is sprintf have %F for the countryOfOrigin value? I would have expected %s or at least some difference to the format since there's an extra field being printed

Reply:
sprintf("%s-%s-%s-%", $id, $sku, $hsCode, $countryOfOrigin);
This returns everything with hypens as delimeters but yet if a sku contains a hypen then it may cause a problem. 
I still don't know where or what is using this function.